docs/knowledge-base/monitoring/Tempo.mdx

```diff
@@ -0,0 +1,24 @@
+---
+id: Tempo
+title: Tempo
+sidebar_label: Tempo
+sidebar_position: 6
+sidebar_class_name: monitoring
+slug: /knowledge-base/monitoring/Tempo
+toc_min_heading_level: 2
+toc_max_heading_level: 5
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Grafana Tempo
+
+You can access Grafana Tempo at https://grafana.mapcolonies.net/datasources/edit/tempo.
+
+Grafana Tempo is a distributed tracing backend used to store and query trace data received from OpenTelemetry Collector or other tracing sources like Jaeger. It allows you to analyze traces for system performance, error detection, and latency tracking. Tempois capable of storing trace data at high throughput without requiring an index, making it highly efficient for large-scale applications.
```

Comment:
OTEL collector and Jaeger are not the same, rephrase the sentence

Reply:
I said it can recive data from OTEL or Jagger for example. It is not the same.